src/extensions/common/SortingUnitPlotGrid.tsx

```diff
@@ -16,7 +16,9 @@ const SortingUnitPlotGrid: FunctionComponent<Props> = ({ sorting, selection, sel
     const sortingInfo: SortingInfo | undefined = useSortingInfo(sorting.sortingObject, sorting.recordingObject)
 
     const visibleUnitIds = selection.visibleUnitIds
-    let unit_ids: number[] = (sortingInfo ? sortingInfo.unit_ids : []).filter(uid => ((!visibleUnitIds) || (visibleUnitIds.includes(uid))));
+    let unit_ids: number[] = (sortingInfo ? sortingInfo.unit_ids : [])
+        .filter(uid => ((!visibleUnitIds) || (visibleUnitIds.includes(uid))))
+        .filter(uid => ((!selection.applyMerges) || (isMergeGroupRepresentative(uid, sorting.curation))))
```

Comment:
If we are applying merges, only show those units that are 'representatives' of the merge group. The representative is the one with the lowest numerical id.